### ISSUE TITLE
Shorthand for adding properties file in ArchaiusModule

### ISSUE THREAD

**Issue by twicksell** [Netflix/archaius#390]
It would be nice to be able to easily specify additional properties when creating an Injector. Something along the lines of:

```
InjectorBuilder.fromModules(new SomeOtherModule(), new ArchaiusModule() {
      @Override
       protected void configureArchaius() {
                 addConfigFromPropertiesFile("foo.properties");
       }
});
```

**Comment by howardyuan** [comment]
As of current Archaius2, the override through any resource can be achieved as following,
            @Override
            protected void configureArchaius() {
                try {
                    bindApplicationConfigurationOverride().toInstance(
                            DefaultConfigLoader.builder().build().newLoader().load("application-override"));
                } catch (ConfigException e) {
                    throw new RuntimeException(e);
                }
            }
        });

**Comment by elandau** [comment]
Solved with #411
